core/src/test/java/org/infinispan/expiration/impl/ExpirationSingleFileStoreDistListenerFunctionalTest.java

```diff
@@ -54,6 +49,12 @@ protected void removeFromContainer(String key) {
       extraCache.getAdvancedCache().getDataContainer().remove(key);
    }
 
+   @Override
+   protected void processExpiration() {
+      super.processExpiration();
+      TestingUtil.extractComponent(extraCache, InternalExpirationManager.class).processExpiration();
```

Comment:
I suggest a comment to explain that `super.processExpiration()` only processes entries primary owned by the first node.

Reply:
Okay.